app/src/main/java/org/p2p/wallet/appsfly/AppsFlyerService.kt

```diff
@@ -19,7 +18,7 @@ class AppsFlyerService(private val context: Context) {
     fun install(application: Application, devKey: String) {
         val appsFlyer = AppsFlyerLib.getInstance()
         kotlin.runCatching {
-            appsFlyer.setDebugLog(BuildConfig.DEBUG)
+            appsFlyer.setDebugLog(false) // turn to true if needed
```

Comment:
feature flag?)😉

Reply:
nah, it's so verbose, all the logcat in this stuff